feature_engine/variable_manipulation.py

```diff
@@ -37,12 +81,27 @@ def _find_numerical_variables(X, variables=None):
         if len(X[variables].select_dtypes(exclude='number').columns) != 0:
             raise TypeError("Some of the variables are not numerical. Please cast them as numerical "
```

Comment:
wonder if we should replace this list comparison with a genexp? Did I not do this in the v1.0.0 branch? If not, maybe we should consider

Reply:
The default logic for ```variables``` is that the user should provide the list of strings. If the user provides only one variable as a string the function raises ```AttributeError``` due to a Pandas logic. Do we need to restrict the function to take the only a list of strings and not a string?
@solegalli

Reply:
Changed list comprehension by genexp